integration/travis.sh

```diff
@@ -12,4 +12,19 @@ sudo mkdir -p /etc/servicebuilder.d /var/service-stage /var/service
 
 sudo cp $GOPATH/bin/p2-exec /usr/local/bin
 
+# make ssl certs
+subj="
+C=US
+ST=CA
+O=SQ
+localityName=SF
+commonName=$HOSTNAME
+organizationalUnitName=Vel
+emailAddress=doesntmatter@something.edu
+"
+CERTPATH=/var/tmp/certs
+mkdir -p $CERTPATH
+openssl req -x509 -newkey rsa:2048 -keyout $CERTPATH/key.pem -out $CERTPATH/cert.pem -nodes -days 300 -subj "$(echo -n "$subj" | tr "\n" "/")"
```

Comment:
I'm not sure. The reason I did it this way is because the certificates only work on hosts who's hostname == the commonName field of the key. Since the travis machines don't all have the same hostname I generated them on the fly. If there was a way to make keys/certs that work everywhere then we could just check them in.